packages/angular/projects/tx-native-angular-sdk/src/lib/interfaces.ts

```diff
@@ -0,0 +1,21 @@
+export interface ITranslationServiceConfig {
+  token: string;
+  secret: string;
+  sourceLocale: string;
+}
```

Comment:
You may check https://github.com/transifex/transifex-javascript/tree/master/packages/native#initialize-library for initialization options on the native SDK. For example sourceLocale is no longer valid and there are more options there.

Reply:
Done